driver-reactive-streams/src/main/com/mongodb/reactivestreams/client/internal/BatchCursorPublisher.java

```diff
@@ -106,6 +106,7 @@ public Publisher<T> first() {
                 .flatMap(batchCursor -> Mono.create(sink -> {
                     batchCursor.setBatchSize(1);
                     Mono.from(batchCursor.next())
+                            .contextWrite(sink.contextView())
```

Comment:
Done

Reply:
> Should the existing contextWrite call on line 119 be removed?

I thought it's better to remove the existing call, and leave the new call that comes right after creating the nested `Mono`, because in all other places `contextWrite` is called right after creating a nested `Mono`.

Reply:
Removed the use of `subscribe` from within the `BatchCursorPublisher`